engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -373,6 +373,18 @@ paths:
           required: true
           type: string
           in: path
+        - name: outputs
+          description: Include output metadata
+          required: false
+          type: boolean
+          in: query
+          default: true
+        - name: timings
```

Comment:
TOL: nickpick if it's not too late... but these might be better referred to as 'events'.  events do have times associated with them, but this makes me think of performance timings etc not executions state transitions